NET01.1Solution/NET01.1Task/VideoMaterial.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NET01._1Task
+{
+    public class VideoMaterial : Entity, IVersionable
+    {
+        public byte[] Version;
+        public Uri VideoUri;
+        public Uri SplashScreenUri;
+
+        public enum VideoFormat
+        {
+            Unknown,
+            Avi,
+            Mp4,
+            Flv
+        };
+
+        public VideoFormat VideoFormatValue { get; set; }
+
+        public VideoMaterial(string description, string urlVideo, string urlSplash, VideoFormat vidFormatVal) : base (description)
+        {
+            VideoUri = Uri.TryCreate (urlVideo, UriKind.Absolute, out VideoUri)? new Uri(urlVideo) : throw new ArgumentNullException(nameof(urlVideo), "Video URL cannot be empty");
```

Comment:
Uri.TryCreate throws exception not only when uri is null, but when it is poorly formatted also. 
Throw InvalidArgumentException instead of ArgumentNullException